environment.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
   - pbkdf2=1.3
   - distro=1.5.0
   - paramiko=2.7.1
+  - mock=4.0.2
```

Comment:
Does this need to be in the yml? It's not technically necessary for running main.py but is necessary for development. If it is included, should the yml also include the libraries needed to run the scripts?

Reply:
well since this file is specifically for development since we use a different script to load everything onto the jetson i see no problem here.